packages/utils/regl-renderer/types/bound-utils/computeBounds.test.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Why a declaration for tests? Can this be removed?

Reply:
Same as above...